dotlottie/src/main/java/com/lottiefiles/dotlottie/core/compose/ui/DotLottieAnimation.kt

```diff
@@ -144,19 +147,34 @@ fun DotLottieAnimation(
         }
     }
 
-    LaunchedEffect(loop, autoplay, playMode, useFrameInterpolation, speed, segments) {
+    LaunchedEffect(
+        loop,
+        autoplay,
+        playMode,
+        useFrameInterpolation,
+        speed,
+        segments,
+        theme,
```

Comment:
I'd suggest to name it `theme_id`
```suggestion
        themeId,
```